### PR TITLE
Fixing issues with Microsoft.Build.xsd scheme

### PR DESCRIPTION
**Pull request by oldmine**
- Added missed xsd include in csproj
- Fixed broken include reference to another xsd scheme

### CODE REVIEW

File: src/MSBuild/Microsoft.Build.xsd [hunk 1]
```diff
@@ -5,7 +5,7 @@ xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
 elementFormDefault="qualified">
 
     <!-- =================== IMPORT COMMON SCHEMA =========================== -->
-    <xs:include schemaLocation="MSBuild\Microsoft.Build.CommonTypes.xsd"/>
+    <xs:include schemaLocation="Microsoft.Build.CommonTypes.xsd"/>
```
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.xsd]: In the [internal source that controls the final VS layout](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=%2Fsrc%2FSetupPackages%2FCoreIde%2FCommunity%2Fres%2Ffiles.swr&version=GBmaster&line=203&lineEnd=211&lineStartColumn=1&lineEndColumn=1&lineStyle=plain), CommonTypes is in a subfolder relative to `Microsoft.Build.xsd`:

```
folder "InstallDir:\Xml\Schemas\$(LCID)"
  file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\Xml\Schemas\DotNetConfig.xsd"
  file source="!(bindpath.sources)\env\msenv\Schemas\snippetformat.xsd"
  file source="!(bindpath.sources)\env\msenv\Schemas\vstemplate.xsd"
  file source="!(bindpath.binaries.x86)\bin\i386\xmake\Microsoft.Build.xsd"
  folder "MSBuild"
    file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\xmake\MSBuild\Microsoft.Build.Commontypes.xsd"
    file source="!(bindpath.binaries.x86)\Localized\$(Lang)\bin\i386\xmake\MSBuild\Microsoft.Build.Core.xsd"
```

So I think this should be reverted and instead the other files should be placed in a subfolder in the MSBuild.csproj output.

File: src/MSBuild/Microsoft.Build.xsd [hunk 2]
```diff
@@ -1,11 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xs:schema targetNamespace="http://schemas.microsoft.com/developer/msbuild/2003" 
 xmlns:xs="http://www.w3.org/2001/XMLSchema"
 xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
 elementFormDefault="qualified">
 
     <!-- =================== IMPORT COMMON SCHEMA =========================== -->
-    <xs:include schemaLocation="MSBuild\Microsoft.Build.CommonTypes.xsd"/>
+    <xs:include schemaLocation="MSBuild/Microsoft.Build.CommonTypes.xsd"/>
```
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.xsd]: Can you revert the changes in this file back to the contents of `master` please?

Comment (oldmine) [reply src/MSBuild/Microsoft.Build.xsd]: Done

**Comment by rainersigwald** [comment]
Thanks @oldmine!
